apis/kubedb/v1alpha2/druid_helpers.go

```diff
@@ -618,3 +683,31 @@ func (d *Druid) ReplicasAreReady(lister pslister.PetSetLister) (bool, string, er
 	}
 	return checkReplicasOfPetSet(lister.PetSets(d.Namespace), labels.SelectorFromSet(d.OffshootLabels()), expectedItems)
 }
+
+func (d *Druid) GetAppBinding(name string, namespace string) (*appcat.AppBinding, error) {
+	appbinding := &appcat.AppBinding{}
+	appbinding.Namespace = namespace
+	appbinding.Name = name
+
+	if err := DefaultClient.Get(context.TODO(), client.ObjectKeyFromObject(appbinding), appbinding); err != nil {
+		klog.Error(err, fmt.Sprintf("failed to get appbinding for metadata storage %s/%s", name, namespace))
+		return nil, err
+	}
+	return appbinding, nil
+}
+
+func (d *Druid) GetMySQLName() string {
+	return d.OffShootName() + "-mysql-metadata"
+}
+
+func (d *Druid) GetPostgresName() string {
+	return d.OffShootName() + "-pg-metadata"
+}
```

Comment:
From operator, call `GetMetadataStorageName()` instead of these two. 
```
func (d *Druid) GetMetadataStorageName() string {
       if d.spec.metadataStorage.Type == postgres {
              return d.OffShootName() + "-pg-metadata"
       }
	return d.OffShootName() + "-mysql-metadata"
}

```